src/components/OutlinedButton/OutlinedButton.tsx

```diff
@@ -24,16 +25,25 @@ export const OutlinedButton: React.FC<IProps> = ({
   className,
   disabled = false,
   startIcon,
-  labelClassName
+  labelClassName,
+  balance
 }) => {
   const classes = useStyles()
+
+  const tokenBalance = isNaN(Number(balance))
+  const disabledButton = disabled && tokenBalance
+
   return (
     <Button
-      className={classNames(classes.general, className)}
+      className={classNames(
+        classes.general,
+        !isNaN(Number(balance)) && classes.activeButton,
```

Comment:
set these styles through className prop; what if this component will be used somewhere, where there will be no such thing as 'balance'?